src/Test/LoggerInterfaceTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Psr\Log\Test;
+
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Provides a base test class for ensuring compliance with the LoggerInterface.
+ *
+ * Implementors can extend the class and implement abstract methods to run this
+ * as part of their test suite.
+ */
+abstract class LoggerInterfaceTest extends TestCase
+{
+    /**
+     * @return LoggerInterface
```

Comment:
```suggestion
     * Return an instance of the logger to be tested.
     * 
     * @return LoggerInterface
```

Better self-documentation.